WTWSMS/common/on_actions/WtWSMS_on_actions.txt

```diff
@@ -73,6 +76,7 @@ on_startup = {
 		
 		beta.1 # Beta status warning
 		
+		misc_event.12
```

Comment:
Why do you have this as on_actions event rather than an effect added in title history of the counties like this:

	effect = {
		location = {
			add_province_modifier = {
				modifier = XXX
				duration = X
			}
		}
	}
?

Reply:
It's simple, in history you can't control the duration like that.

Reply:
How so? There is a duration.

Reply:
Whenever you start, the duration is the same, which won't work well.

Reply:
Why would putting 10 blocks of the code you wrote above for all provinces be better than one maintenance event fired on start?

Reply:
No, see how I scripted my event.